pilot/pkg/model/service.go

```diff
@@ -271,6 +271,35 @@ type ServiceInstance struct {
 	Endpoint    *IstioEndpoint `json:"endpoint,omitempty"`
 }
 
+// ServiceTarget includes a Service object, along with a specific service port
+// and target port. This is basically a smaller version of ServiceInstance,
+// intended to avoid the need to have the full object when only port information
+// is needed.
+type ServiceTarget struct {
+	Service *Service
+	Port    ServiceInstancePort
+}
+
+type (
+	ServicePort = *Port
```

Comment:
The only reason to do so is to make the embedded type have a different name so it's more clear

Reply:
Does `ServicePort = *Port` allow `*ServicePort` to be the same as `*Port`? Assuming no, but if that's how it worked it would be more readable instead of making it less obvious that there is a pointer here.